### PR TITLE
Single Page App

### PR DESCRIPTION
**Pull request by jimitpatel3699**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+                        </a>
+                    </div>
+                    <div class="nav-menu col-xs-12 col-sm-10">
+                        <div class="menu-main-menu-container">
+                            <ul id="menu-main-menu" class="menu">
+                                <li id="menu-item-403" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-403"><a href="#home">Home</a></li>
+                                <li id="menu-item-1149" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1149"><a href="#imageSlider">Images</a></li>
+                                <li id="menu-item-2321" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2321"><a href="#services">Services</a></li>
+
+                                <li id="menu-item-1149" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1149"><a href="#About">About_Us</a></li>
```
Comment (utkarshsimform) [inline index.html]: Why _ in heading?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+                        </h3>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!--body image slider-->
+    <div class="container my-7" id="imageSlider">
+        <hr class="rule-lg" />
+        <h4 class="text-center"><u>Image Slider </u></h4>
```
Comment (utkarshsimform) [inline index.html]: Give some proper heading

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (utkarshsimform) [inline .vscode/settings.json:1]: Make SPA Responsive for Desktop, Laptop, Mobile

**Review by utkarshsimform — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (jimitpatel3699) [reply index.html]: Resolved

File: index.html (hunk 2 above)
Comment (jimitpatel3699) [reply index.html]: Resolved

File: .vscode/settings.json (hunk 3 above)
Comment (jimitpatel3699) [reply .vscode/settings.json:1]: Resolved

**Review by utkarshsimform — APPROVED** [review]

**Pull request merged**